BlazarUI/app/scripts/components/dashboard/StarredModulesTableRow.jsx

```diff
@@ -29,41 +39,43 @@ class StarredModulesTableRow extends Component {
       );
     }
 
-    const latestBuild = item.builds[0].build;
-    const latestBuildGitInfo = item.builds[0].gitInfo;
-
+    const commitInfo = latestBuild.get('commitInfo');
     let commitMessage;
     let sha;
 
-    if (has(latestBuild, 'commitInfo')){
+    if (commitInfo !== undefined) {
       commitMessage = (
-        <CommitMessage message={latestBuild.commitInfo.current.message} />
+        <CommitMessage message={commitInfo.get('current').get('message')} />
       );
     }
     
-    if (latestBuild.sha !== undefined) {
+    if (latestBuild.get('sha')) {
       const gitInfo = {
-        host: latestBuildGitInfo.host,
-        organization: latestBuildGitInfo.organization,
-        repository: latestBuildGitInfo.repository
+        host: latestBuildGitInfo.get('host'),
+        organization: latestBuildGitInfo.get('organization'),
+        repository: repository
       }
 
+      latestBuild.sha = latestBuild.get('sha');
```

Comment:
So, are all of these `get`s a necessary consequence of using `ImmutableMap`s? It worked well, save for this line where I need to force `sha` as a field in `latestBuild`, since the `Sha` component doesn't expect an `ImmutableMap` (and other components that instantiate a `Sha` do not pass an `ImmutableMap`)